aio/protractor.conf.js

```diff
@@ -17,6 +17,11 @@
 
 const {SpecReporter} = require('jasmine-spec-reporter');
 
+console.log('=======================================')
+console.log(!!process.env.TRAVIS && process.env.TRAVIS_BRANCH === 'master')
```

Comment:
Do we need this?

Reply:
I have found additional issue and had to "debug" travis :slightly_smiling_face: